keel-front50/src/main/kotlin/com/netflix/spinnaker/keel/front50/model/Application.kt

```diff
@@ -1,13 +1,26 @@
 package com.netflix.spinnaker.keel.front50.model
 
+import com.fasterxml.jackson.annotation.JsonAnyGetter
+import com.fasterxml.jackson.annotation.JsonAnySetter
+
+/**
+ * A Spinnaker application, as represented in Front50.
+ */
 data class Application(
   val name: String,
   val email: String,
+  val createTs: String,
   val dataSources: DataSources?,
   val repoProjectKey: String? = null,
   val repoSlug: String? = null,
-  val repoType: String? = null
-)
+  val repoType: String? = null,
+  @get:JsonAnyGetter val details: MutableMap<String, Any?> = mutableMapOf()
```

Comment:
The `details` field is a "catch-all" map of all non-explicitly-mapped properties that appear in the JSON.